server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt

```diff
@@ -89,6 +90,11 @@ class KotlinLanguageServer : LanguageServer, LanguageClientAware, Closeable {
         serverCapabilities.executeCommandProvider = ExecuteCommandOptions(ALL_COMMANDS)
         serverCapabilities.documentHighlightProvider = Either.forLeft(true)
 
+        val db = setupServerDatabase(params)
```

Comment:
Hm, we need the database initialization to happen after the `initialize`, since the client supplies the `storagePath`, right? It would be neat if we wouldn't need to wire up the database lazily and could pass it using the usual constructor injection pattern, but I don't really see a better way to do this either...